doc/news/changes/minor/20200330MartinKronbichler

```diff
@@ -0,0 +1,6 @@
+Fixed: FEValues would previously detect cell similarities when combined with
+MappingFEField when the underlying mesh had similarities, even though the
+deformed cells did not. This would lead to wrong Jacobians and transformed
+shape function gradients, among others, when running deal.II without threads.
+This is now fixed.
```

Comment:
Can you update that the same fix applies to `MappingManifold` (since you also do this in this patch)?